Objects/longobject.c

```diff
@@ -2551,14 +2554,12 @@ long_divrem(PyLongObject *a, PyLongObject *b,
         (size_a == size_b &&
          a->ob_digit[size_a-1] < b->ob_digit[size_b-1])) {
         /* |a| < |b|. */
-        *pdiv = (PyLongObject*)PyLong_FromLong(0);
-        if (*pdiv == NULL)
-            return -1;
         *prem = (PyLongObject *)long_long((PyObject *)a);
         if (*prem == NULL) {
-            Py_CLEAR(*pdiv);
             return -1;
         }
+        *pdiv = (PyLongObject*)_PyLong_Zero;
+        Py_INCREF(_PyLong_Zero);
```

Comment:
Whenever possible, even if we are protected by the GIL, I prefer to have Py_INCREF before assignement, for consistency.